### ISSUE TITLE
Charts on address pages fails to load

### ISSUE THREAD

**Issue by jholdstock** [decred/dcrdata#758]
1. Visit the page for an address, eg: https://explorer.dcrdata.org/address/DsjZTtatQUatzhUGdwHtdCB24R9yMC7JpY3
2. Click the "Charts" button
3. Click the "Zoom" > "Day" button

Infinite spinning wheel is displayed, and a Javascript error is printed to the console:

```
stimulus.umd.js:5 
Error invoking action "click->address#onZoom"

TypeError: Cannot read property 'resetZoom' of undefined
    at e.onZoom (address.js:226)
    at t.invokeWithEvent (stimulus.umd.js:5)
    at t.handleEvent (stimulus.umd.js:5)

{identifier: "address", controller: e, element: div.container.main, event: MouseEvent}
```

**Comment by buck54321** [comment]
It looks like it's just a missing check for an initialized chart.

Other than hiding the controls until the chart is initialized, should be find to just add

```
if (this.graph == undefined) {
    return
}
```
at line 225 in public/js/controllers/address.js

I'm sort of new to github, so I'm not certain whether I should create a pull request on an issue that has been assigned. I don't want to step on anybody's toes.

**Comment by dmigwi** [comment]
@buck54321 Sure please do. I couldn't exactly replicate the bug above but if you have found a fix please create a PR for it.

**Comment by chappjc** [comment]
Resolved by PR #760
